iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -235,17 +213,55 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     :return: a dictionary of labels to set on the bundle
     :rtype: dict
     """
-    package_name, labels = _apply_package_name_suffix(metadata_path, organization)
-
     try:
         operator_manifest = OperatorManifest.from_directory(manifests_path)
     except (ruamel.yaml.YAMLError, ruamel.yaml.constructor.DuplicateKeyError) as e:
         error = f'The Operator Manifest is not in a valid YAML format: {e}'
         log.exception(error)
         raise IIBError(error)
 
-    found_pullspecs = set()
-    operator_csvs = []
+    conf = get_worker_config()
+    organization_customizations = conf['iib_organization_customizations'].get(organization, [])
+
+    annotations_yaml = _get_package_annotations(metadata_path)
+    package_name = annotations_yaml['annotations'][
+        'operators.operatorframework.io.bundle.package.v1'
+    ]
+    labels = {}
+    if organization_customizations:
```

Comment:
Optional: You could default `organization_customizations` to something like this and then you wouldn't need the if/else conditional.
```
[
    {'type': 'package_name_suffix'},
    {'type': 'registry_replacements'},
    {'type': 'csv_annotations'},
]
```